.github/workflows/ci.yml

```diff
@@ -25,3 +25,26 @@ jobs:
       - run: yarn install --frozen-lockfile
       - run: yarn prettier --check .
       - run: yarn run lint
+      - name: Set up PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: 7.4
+          tools: composer
+      - name: Create Acquia CMS project
+        run: composer create-project acquia/drupal-recommended-project acms
+      - name: Copy Drush configuration
+        run: |
+          mkdir $HOME/.drush
+          cp drush.yml $HOME/.drush
+      - name: Install Acquia CMS and start web server
+        run: |
+          ./vendor/bin/acms acms:install acquia_cms_headless --no-interaction
+          ./vendor/bin/drush php:script ../init-ci.php > ../starters/basic-starter/.env.local
+          ./vendor/bin/drush serve --quiet 8080 &
```

Comment:
I'm surprised if this will work properly because we've had quite a few reports from folks that this has not worked at all.

Reply:
What, `drush serve`? I thought about it and I figured it should work as long as the *backend* isn't making subrequests.

Reply:
I further figured, since the backend is basically an unmodified Acquia CMS, it's not likely they're making subrequests unless there are specific code paths that does it. I don't know much about JSON:API, but if it doesn't use subrequests to generate its own output, then I would imagine we're pretty safe.